### DISCUSSION TITLE
Button Node (Beta)

### DISCUSSION THREAD

**Discussion by zachowj** [zachowj/node-red-contrib-home-assistant-websocket#546] [General]
Added with version 0.41.0

**Comment by Sebazzz** [comment]
That would be awesome :) Incorrectly opened FR at the add-on project: https://github.com/hassio-addons/addon-node-red/issues/1227

**Comment by marleyjaffe** [comment]
I think the field "Friendly Name" should be updated to "Entity ID". This would provide more clarity when entering information. No other issues so far!

**Reply by zachowj** [reply]
It's really only the entity id on the first deployment after that it only changes the friendly name.

https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/custom_integration/#tip
